tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -313,6 +315,7 @@ private DAGRequest buildRequest(DAGRequest.Builder dagRequestBuilder) {
    * @return final DAGRequest built
    */
   private DAGRequest.Builder buildScan(boolean buildIndexScan) {
+    outputOffsetsToBeAdded.clear();
```

Comment:
is there any reason that this outputOffsets should be used externally since it is local? will there be any concurrent issue?

Reply:
done